documentation.css

```diff
@@ -0,0 +1,100 @@
+html {
+  scroll-behavior: smooth;
+  font-size: 62.5%;
+}
+
+body {
+  color: var(--primary-font-color);
+}
+
+.wrapper {
+  display: flex;
+  position: relative;
+}
+.wrapper .sidebar {
+  width: 20%;
+  background: var(--primary-background-color);
+  padding: 1.5rem;
+  overflow-x: auto;
+  position: sticky;
+  top: 0rem;
+  height: calc(100vh - 2rem);
+}
+.wrapper .sidebar h2 {
+  margin-bottom: 1rem;
+  margin-top: 1rem;
+  font-size: 2rem;
+  font-weight: 700;
+  letter-spacing: 1px;
+}
+.wrapper .sidebar ul li {
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+  border-top: 1px solid rgba(48, 41, 41, 0.05);
+}
+.wrapper .sidebar ul li a {
+  padding: 1rem 1.5rem;
+  display: block;
+  color: var(--primary-font-color);
+  font-size: 1.5rem;
+}
+.wrapper .sidebar ul li:hover a {
+  color: var(--secondary-font-color);
+  background: var(--primary-color);
+  border-radius: 5px;
+}
+.wrapper .sidebar .socila-media {
+  align-items: center;
+  justify-content: center;
+}
+.wrapper .sidebar .socila-media a {
+  color: var(--primary-font-color);
+  font-size: 2rem;
+  padding: 0 1.5rem;
+  line-height: 5rem;
+  text-align: center;
+}
+.wrapper .sidebar .socila-media a:hover {
+  color: var(--primary-color);
+}
+
+.wrapper .main-documentation {
+  width: 80%;
+  padding: 2rem;
+  font-size: 1.6rem;
+}
+.wrapper .main-documentation {
+  width: 80%;
+  padding: 2rem;
+  font-size: 1.8rem;
+  line-height: 1.5;
+}
+.wrapper .main-documentation .component-intro {
+  margin-bottom: 1.5rem;
+  font-size: 2.2rem;
+  font-weight: 400;
+  margin-top: 1rem;
+}
+.wrapper .main-documentation .component-intro h1 {
```

Comment:
Try to not chain classes like this due to specificity issues that may arise. Its regarded as bad practice.